tests/flow/require/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -165,7 +165,7 @@ E.stringValue; // Error: The E exports obj has no \'stringValue\' property
 var a = \"./E\";
 require(a); // Error: Param must be string literal
 require(\`./E\`); // template literals are ok...
-require(\`\${\"./E\"}\`); // error: but only if they have no expressions
+require(\`\${\'./E\'}\`); // error: but only if they have no expressions
```

Comment:
I guess I would have preferred prettier to consistently have enforced double quotes everywhere, but I guess we can't selectively enable and disable features ...?